pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: tinycolor2
 description: Flutter Color manipulation and conversion, ported from JS tinycolor2
-version: 3.0.0
+version: 3.0.1
 homepage: https://github.com/TinyCommunity/tinycolor2
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  flutter: ">=3.0.0 <4.0.0"
```

Comment:
https://github.com/TinyCommunity/tinycolor2/actions/runs/3065842779/jobs/4950358056

@WieFel looks like they don’t want upper restraint for flutter packages